lisa/sut_orchestrator/azure/features.py

```diff
@@ -500,6 +500,12 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._initialize_information(self._node)
 
 
+class ACC(AzureFeatureMixin, features.ACC):
```

Comment:
If there is no special logic, it doesn't need to create it in platform.

Reply:
I see a similar thing being done for `class Nvme(AzureFeatureMixin, features.Nvme):`, so I thought this is the norm we followed. Can you please clarify?

Reply:
It's optional. Can you try if it works, if so, we need to remove nvme as well.

Reply:
Verified that removing NVME doesn't cause any issue